apps/typegpu-docs/astro.config.mjs

```diff
@@ -87,6 +87,10 @@ export default defineConfig({
               label: 'Generating JS from WGSL',
               slug: 'guides/generating-js-from-wgsl',
             },
+            {
+              label: 'TypeGPU CLI',
+              slug: 'guides/tgpu-cli',
```

Comment:
we could add the "new" badge like for the Roots guide to make it more visible for now